include/GafferScene/Light.h

```diff
@@ -40,6 +40,7 @@
 #include "GafferScene/ObjectSource.h"
 
 #include "IECoreScene/ShaderNetwork.h"
+#include "Gaffer/CompoundDataPlug.h"
```

Comment:
Include order

Reply:
FIxed in https://github.com/GafferHQ/gaffer/pull/3519/commits/b695884a3903ff41d90c2fc557c056a3cff075dc